config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ if (NOT CHIP_LIBRARIES)
 endif()
 
 if (CONFIG_CHIP_LIB_SHELL)
-    list(APPEND CHIP_LIBRARIES ${CMAKE_CURRENT_BINARY_DIR}/obj/third_party/connectedhomeip/src/lib/shell/lib/libCHIPShell.a)
+    list(APPEND CHIP_LIBRARIES ${CMAKE_CURRENT_BINARY_DIR}/lib/libCHIPShell.a)
```

Comment:
-lCHIPShell should be enough since we add ${CMAKE_CURRENT_BINARY_DIR}/lib to search directories.